### PR TITLE
fix(curriculum): Split Cat App Step 12 into two parts

### LINKED ISSUE #47010: CatPhotoApp Step 12 is not clear
There are many users every day having issues with this step.

I have no idea what could be changed, but something would need to be changed

### PR DESCRIPTION
**Pull request by jeremylt**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47010

This is the non-controversial half of #47149

### CODE REVIEW

**Comment by jeremylt** [comment]
Sorry for the noise, I was editing via the web editor.

This seems to work when I test it via Gitpod.

**Review by bbsmooth — COMMENTED** [review]
Do we care if they delete the "cat photos" link below the `p` element? I can definitely see them "moving" the link by thinking they need to delete it even though we aren't telling them to. And then if they do delete it in this step and it reappears in the next step is that a problem we need to be concerned about?

**Comment by ieahleen** [comment]
Note that when you build the site, even on gitpod, there is a challenge editor that makes this much easier. See at: https://contribute.freecodecamp.org/#/how-to-work-on-practice-projects?id=using-the-challenge-editor

I've restarted the tests

**Comment by jeremylt** [comment]
> Do we care if they delete the "cat photos" link below the `p` element? I can definitely see them "moving" the link by thinking they need to delete it even though we aren't telling them to. And then if they do delete it in this step and it reappears in the next step is that a problem we need to be concerned about?

Honestly? I'm not sure. I think that users who look at the forum for old hint may try to do it all in one step. I could go either way here.

**Comment by Tracerract** [comment]
I've helped a couple people on the forums on this step specifically, and I've noticed something else about this step that seems to be confusing some people. Up until this point, they've been nesting one thing entirely inside another, like:
```
<html>
     <body>
     <h1>CatPhotoApp</h1>
     </body>
</html>
```
And that nesting has been the same for their text. Each sentence has been contained entirely within one set of tags. This step is the first time they need to wrap one specific part of the sentence in tags, instead of the entire thing. What I was noticing is that it seemed like not everyone understood that some tags can just go in the middle of text, to only apply to specific words of that text. I feel like if we change the way the directions are phrased, we'd be able to communicate this at least a little more efficiently.
Rather than describing it like:
' by adding a new anchor element.'
It could be described as:
' by starting a new anchor tag just before the word "cat" and placing the closing tag after the word "photos" and before the period.'
I know that's a little verbose, but anything that describes it more as wrapping the already existing words in an element, rather than just adding a new element inside the 'p' element, should help the people running into this.

**Comment by jeremylt** [comment]
That is way, way too verbose in my opinion.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md
Comment (jeremylt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md:12]:
```suggestion
In the text of your `p` element, turn the words `cat photos` into a link to `https://freecatphotoapp.com` by nesting these words inside of a new anchor element.
```

**Comment by Sboonny** [comment]
@jeremylt Is this ready for review?

**Comment by jeremylt** [comment]
It's been ready for review for a while. I'm not sure why staff removed the ready for review tag.

**Review by Sboonny — DISMISSED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md
Comment (ieahleen) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md:16]:
```suggestion
You should nest a new anchor (`a`) element within the `p` element.
```

Considering that the step has changed, it makes sense to adapt the tests. Otherwise we risk to not alleviate the confusion.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62dabe2ef403a12d5d295273.md
Comment (ieahleen) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62dabe2ef403a12d5d295273.md:14]:
```suggestion
Your code should only have one anchor (`a`) element.
```

OR

```suggestion
Your code should contain only one anchor (`a`) element.
```

or something

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62dabe2ef403a12d5d295273.md
Comment (ieahleen) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62dabe2ef403a12d5d295273.md:20]:
```suggestion
The `p` element should still have the anchor (`a`) element inside of it.
```

**Review by Sboonny — DISMISSED** [review]

**Review by moT01 — DISMISSED** [review]
One minor suggestion - other than that, this LGTM @jeremylt 🎉 I like splitting this up into two steps.

I'm going to block this for a moment - I want to see what the process is when we add a new challenge. I think it might block i18n builds until the new file gets downloaded from crowdin, so I just want to ask the staff about that real quick.

**Comment by ieahleen** [comment]
Yes, it blocks the builds, but we are pretty quick - at this time only three languages have the responsive web design new enabled

**Comment by moT01** [comment]
Sorry for the delay here @jeremylt - I'm not sure I actually got the answer I was looking for. I was told...

> Wait for the translations to be added to the new challenge - the PR will sync daily and the action will stop failing when translations catch up

Which seems to imply that the translations will be added to this PR. But I'm not sure how that works - I dunno, maybe they will. @ieahleen, can you shed some light on this for me? Do we want to wait for something like that or merge this with just the English and block deploys?

**Comment by ieahleen** [comment]
@moT01 the new files can't be imported in crowdin until this is merged. Once it is merged, it will be imported into crowdin.
The crowdin PR will fail initially, and sync daily on its own, until the translation is complete, and the translated files are downloaded into the crowdin PR, which then can be merged.

Naomi has created the scripts, and we have everything documented, for more info.

**Review by Sboonny — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM @jeremylt 🎉 Thanks for your patience on this.

**Comment by hutong9** [comment]
您好：    双换感谢您的来信，收到信后我会第一时间回复您，如需紧急处理请致电15809253357，谢谢！

**Pull request merged**
